src/forms/calendarComponent.story.jsx

```diff
@@ -70,65 +74,56 @@ storiesOf('Forms/CalendarComponent', module)
 					name="event_time"
 					label="Start at"
 					helperText="Lorem Ipsum is simply dummy text"
-					value={new Date()}
+					value={jodaDate}
 				/>
 			</div>
 		);
 	})
 	.add('sets a valid date range', () => {
-		const min = new Date(),
-			max = new Date(),
-			now = Date.now();
-		min.setDate(min.getDate() - 2);
-		max.setDate(max.getDate() + 7);
-
-		const opts = {
-			allowInput: true,
-			minDate: min,
-			maxDate: max,
-		};
 		return (
 			<div className="span--50">
 				<CalendarComponent
 					name="event_time"
 					label="From 2 days ago to 1 week from now"
-					value={now}
-					datepickerOptions={opts}
+					value={jodaDate}
+					datepickerOptions={{
+						allowInput: true,
+						minDate: jodaDate.minusDays(2),
+						maxDate: jodaDate.plusDays(7),
```

Comment:
I don't think this will work - does it? the `datepickerOptions` are values passed straight to flatpickr, which doesn't support js-joda objects

Reply:
It shouldn't, I hadn't tried the validation on this yet but it would make sense to be a necessary conversion.

Reply:
This will lead to an interesting interface, all of our dates take `js-joda` objects and all of the functionality extended from `flatpicker` wants a `Date` object. We do the conversion for the value in the component, would we want to wrap date range and all of this functionality as well?

Reply:
Hmm tough call. I lean toward _not_ trying to wrap the `options` offered by flatpickr - they're reasonably well-documented [here](https://flatpickr.js.org/options/) and I think treating that as a single source of truth for that config is simplest - e.g. I don't think people would assume that there was a `minDate` option available and then assume that it took a `LocalDate` value without having seen the docs, and once they see the docs, they should know the arg type. I supposed we could change the name of `datepickerOptions` to `flatpickrOptions`, although the flow type definitions are already specified [here](https://github.com/meetup/meetup-web-components/blob/30a537eb04b961f1e3e0b43902a45bcae61bd8b6/src/forms/CalendarComponent.jsx#L12) so once those are exported correctly, consumers should be forced into correct usage... in theory...